recipes/dymos/run_test.py

```diff
@@ -0,0 +1,33 @@
+import os
+import subprocess
+import sys
+from os.path import join, dirname
+
+import dymos
+
+
+test_files_to_delete = [
+    # can't test these, yet, because of playwright
+    ["visualization", "linkage", "test", "test_gui.py"],
+    ["visualization", "linkage", "test", "linkage_report_ui_test.py"],
+    # unsure why this test is failing, but skipping it for now
```

Comment:
Looks like it is opening a matplotlib window and blocking the test until it exits.
You can set the backend via environment variable like `set MPLBACKEND=Agg`. Maybe that would fix some things?

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=682461&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=2fb5b0a7-737c-5f5c-8f3f-f6db6174bacf&l=3366

Reply:
doesn't explain why linux and mac are passing though

Reply:
Thanks Joel!  Testing it in [2fd1d4c](https://github.com/conda-forge/staged-recipes/pull/22430/commits/2fd1d4c09c933cbe398960188f36164946808abd)